src/App.tsx

```diff
@@ -1,7 +1,11 @@
+import { Navigate, Routes, Route } from 'react-router-dom';
 import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
 
 import './App.scss';
+const HomePage = () => <h1 className="title">Home Page</h1>;
+
+const NotFoundPage = () => <h1 className="title">Page not found</h1>;
```

Comment:
It is not a good idea to keep more than one component per file
create a separate file for this components